### PR TITLE
Starter template for the Dagster University project

### PR DESCRIPTION
**Pull request by tacastillo**

### CODE REVIEW

File: examples/project_dagster_university_start/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.dagster]
+module_name = "dagster_university"
```
Comment (tacastillo) [inline examples/project_dagster_university_start/pyproject.toml:6]: NTS on next push: update this and other references to `project_dagster_university`

File: examples/project_dagster_university_start/setup.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="dagster_university",
+    packages=find_packages(exclude=["dagster_university_tests"]),
+    install_requires=[
+        "dagster==1.3.*",
```
Comment (erinkcochran87) [inline examples/project_dagster_university_start/setup.py:7]: Should this be 1.4.*?

File: examples/project_dagster_university_start/setup.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        "dagster-gcp",
+        "dbt-duckdb",
+        "geopandas",
+        "kaleido",
+        "pandas",
+        "plotly",
+        "psycopg2-binary",
+        "shapely"
+    ],
+    extras_require={"dev": ["dagit", "pytest"]},
```
Comment (erinkcochran87) [inline examples/project_dagster_university_start/setup.py:20]: Should `dagit` be `dagster-webserver`?

File: examples/project_dagster_university_start/data/requests/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Requests
+
+This directory simulates a request system in the sensors portion of Dagster University. By creating a "request" file in this directory, your Dagster project will automatically pick up the request and run the pipeline with the parameters specified in the request.
+
+The pipeline stored the results in the `data/results` directory.
```
Comment (PedramNavid) [inline examples/project_dagster_university_start/data/requests/README.md:5]: `stores`?

**Review by PedramNavid — APPROVED** [review]
Love it. Just one question on a word-tense, but overall looks great.

**Review by erinkcochran87 — APPROVED** [review]

**Review by jamiedemaria — COMMENTED** [review]
so cool to see this coming together!!

**Pull request merged**